inference.py

```diff
@@ -221,42 +152,21 @@ def main(params: dict):
     """
     # SET BASIC VARIABLES AND PATHS
     since = time.time()
-
-    debug = get_key_def('debug_mode', params['global'], False)
-    if debug:
-        warnings.warn(f'Debug mode activated. Some debug features may mobilize extra disk space and cause delays in execution.')
-
-    num_classes = params['global']['num_classes']
     task = params['global']['task']
-    num_classes_corrected = add_background_to_num_class(task, num_classes)
-
+    img_dir_or_csv = params['inference']['img_dir_or_csv_file']
```

Comment:
If we only provide a path to a folder, does that means that only tifs will be read (i.e. no gpkg of potential benchmark dataset)?  

If so, we should include this behavior in the README, in the inference section. :)

Reply:
Yes, that is the current implementation. I would document this behaviour in the README.md. tnx